SPECS/python-optree/python-optree.spec

```diff
@@ -30,18 +32,28 @@ A PyTree is a recursive structure that can be an arbitrarily nested Python conta
 
 
 %build
+
+export FFLAGS=$(echo $FFLAGS | sed 's/\(-Wp,-D_GLIBCXX_ASSERTIONS\)//g')
```

Comment:
Aren't we're missing the `%check` section? I understood we caught the issue when you were adding a simple tests, which tried to initialize a tree with an empty tuple.

Reply:
We caught it from the smoke test for tensorflow container. when running %pytest, it complains about missing helper files.  Looks like test provided by python-optree doesn't work properly.